common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1134,6 +1134,7 @@ public final class AdminConstants {
     public static final String E_HOSTNAME = "hostname";
     public static final String E_LIFETIME = "lifetime";
     public static final String E_MESSAGE = "message";
+    public static final String E_USE_DEVICE_ONLY = "usedeviceonly";
```

Comment:
the change we are handling here is to allow this request by device id only too, so the name `E_BY_DEVICEID_ONLY` & `bydeviceidonly` would be appropriate

Reply:
done